src/pspm_dcm_inv.m

```diff
@@ -126,9 +126,9 @@
 try model.meanSCR; catch, model.meanSCR = 0; end
 % These parameters were set with default fallback values but will be
 % determined later by processing (same to pspm_dcm)
-%try model.fixevents; catch, warning('model.fixevents not defined.'); end
-%try model.flexevents; catch, warning('model.flexevents not defined.'); end
-%try model.missing_data; catch, warning('model.missing_data not defined.'); end
+try model.fixevents; catch, warning('model.fixevents not defined.'); end
+try model.flexevents; catch, warning('model.flexevents not defined.'); end
```

Comment:
@dominikbach I do not quite remember why these checking codes are commented. I think they are used as
`.fixevents`: line 341, 403.
`.flexevents`: line 336, 337, 338, 394, 395, 399.
Therefore I have re-enabled them.